test/sanity/exceptionTestCase/testMessageNotUnderstoodWithParams.wtest

```diff
@@ -1,5 +1,5 @@
 class A { 
-	method m1() { throw new Exception("hello you see") }
+	method m1() { throw new Exception() }
```

Comment:
The sanity tests are being completely completely completely changed. 
But... good catch! 
I had the same error in my working branch. I fixed it.

Reply:
Great. I thought this might be a waste of time since you are working on these, but mentioned just in case.